### PR TITLE
Adding touch-action utility

### PR DESCRIPTION
**Pull request by MatteoGauthier**
Hey guys, i added the `touch-action` css property to tailwind :D

### CODE REVIEW

**Review by adamwathan — CHANGES_REQUESTED** [review]
Hey thanks for this one, had this on my list to add soon so cool to have it come in as a PR! Left a few bits of feedback for things that need to change but generally looks good, thanks again.

File: src/plugins/index.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ export { default as flexShrink } from './flexShrink'
 export { default as flexGrow } from './flexGrow'
 export { default as tableLayout } from './tableLayout'
 export { default as borderCollapse } from './borderCollapse'
+export { default as touchAction } from './touchAction'
```
[the remark this answers is not in the document]
Comment (MatteoGauthier) [reply src/plugins/index.js:35]: Hey, thank you for your review. I will change that 👌

File: tests/jit/basic-usage.test.html [hunk 2]
```diff
@@ -145,5 +145,6 @@
     <div class="break-words"></div>
     <div class="z-30"></div>
     <div class="content-none"></div>
+    <div class="touch-manipulation"></div>
   </body>
```
[the remark this answers is not in the document]
Comment (MatteoGauthier) [reply tests/jit/basic-usage.test.html:149]: Yes of course (i forgot to disable headwind vscode extension)

File: stubs/defaultConfig.stub.js [hunk 3]
```diff
@@ -693,6 +693,18 @@ module.exports = {
...
+      'none': 'none',
+      'pan-x': 'pan-x',
+      'pan-left': 'pan-left',
+      'pan-right': 'pan-right',
+      'pan-y': 'pan-y',
+      'pan-up': 'pan-up',
+      'pan-down': 'pan-down',
+      'pinch-zoom': 'pinch-zoom',
+      'manipulation': 'manipulation',
+    },
```
[the remark this answers is not in the document]
Comment (MatteoGauthier) [reply stubs/defaultConfig.stub.js]: hey, i removed lines you've noticed :D

**Comment by MatteoGauthier** [comment]
@adamwathan I tried to correct issues you noticed 📍

**Comment by MatteoGauthier** [comment]
I don't know why the `test` fail..

**Comment by sachinraja** [comment]
I think the problem is that you didn't regenerate the fixtures after moving the import position. Looks like you'll also have to change the class's position in `tests/jit/basic-usage.test.css`. Tests pass for me after addressing both of these issues.

So you need to:
1. `npm run rebuild-fixtures`
2. move the `touch-manipulation` class to right after `cursor-pointer` in `tests/jit/basic-usage.test.css`.

**Comment by MatteoGauthier** [comment]
I don't know why the tests still failing..

**Comment by adamwathan** [comment]
Hey! We refactored the living hell out of the codebase since the time you opened this, so I just opened a new PR and included you as a co-author just to get this merged in quickly 👍🏻 

https://github.com/tailwindlabs/tailwindcss/pull/5603

Thanks a ton man!

**Comment by MatteoGauthier** [comment]
> Hey! We refactored the living hell out of the codebase since the time you opened this, so I just opened a new PR and included you as a co-author just to get this merged in quickly 👍🏻
> 
> #5603
> 
> Thanks a ton man!

Thanks Adam ! Have a good day :D

**Pull request closed without merging**